pkg/skaffold/build/gcb/cloud_build.go

```diff
@@ -290,15 +290,15 @@ func getBuildID(op *cloudbuild.Operation) (string, error) {
 	return buildMeta.Build.Id, nil
 }
 
-func (b *Builder) getDigest(cb *cloudbuild.Build, defaultToTag string) (string, error) {
+func (b *Builder) getDigest(cb *cloudbuild.Build, defaultToTag string, platforms platform.Matcher) (string, error) {
 	if cb.Results != nil && len(cb.Results.Images) == 1 {
 		return cb.Results.Images[0].Digest, nil
 	}
 
 	// The build steps pushed the image directly like when we use Jib.
 	// Retrieve the digest for that tag.
 	// TODO(dgageot): I don't think GCB can push to an insecure registry.
-	return docker.RemoteDigest(defaultToTag, b.cfg)
+	return docker.RemoteDigest(defaultToTag, b.cfg, util.ConvertToV1Platform(platforms.Platforms[0]))
```

Comment:
Hi @tejal29. This line is causing a panic when there are no platforms specified. I'm hoping you can add some context to the open issue #7587 or suggest a fix? Thanks!